proxy/director.go

```diff
@@ -22,24 +22,17 @@ import (
 	"time"
 )
 
-const (
-	// amount of time an endpoint will be held in a failed
-	// state before being reconsidered for proxied requests
```

Comment:
I think we should still keep the second sentence. (move the whole comment to configuration.md).